src/components/common/index.css

```diff
@@ -60,3 +63,51 @@
 .rightMargin {
   margin-right: 2.5rem;
 }
+
+.navbar {
+  overflow: hidden;
+}
+
+.langPic {
+  width: 60px;
+}
+
+.header {
+  margin-top: 0px;
+}
+
+.langDropdown button {
+  background-color: transparent !important;
+  border-color: transparent !important;
+}
+
+.dropdown-toggle::after {
+  display: none !important;
+}
+
+.dropdown-menu.show {
+  right: 20px !important;
+  left: auto;
+  position: absolute;
+  width: 100px;
+  margin: 0px;
+  top: 100px;
+  background-color: white;
+  z-index: 9999;
+  padding: 20px 0;
+}
+
+.dropdown,
```

Comment:
if these are bootstrap classes, you also need to scope them by class, and add a comment to clarify this. now no one else in the app can use dropdown unless it has the same css rules applied :)